num_masks: 1
mask_1:
  low: [0, 6, 62]
  high: [120, 222, 255]
